config/exports/exports.yml

```diff
@@ -149,10 +149,14 @@ hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare:
   csv: hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare
   workflow: scRNA Core PBMC Bank
   plate_includes:
-    - wells.downstream_tubes
     - wells.aliquots
     - wells.aliquots.sample
+    - wells.aliquots.sample.component_samples
     - wells.aliquots.sample.sample_metadata
+    - wells.downstream_tubes
+    - wells.qc_results
+    - wells.transfer_requests_as_target.source_asset
+    - wells.upstream_tubes
```

Comment:
Why are you modifying the query for the `hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare` file?